src/stirling/source_connectors/socket_tracer/uprobe_manager.h

```diff
@@ -334,6 +335,103 @@ class UProbeManager {
           },
       });
 
+  // grpc-c percpu arrays.
+  static constexpr char kGrpcCVersionsName[] = "grpc_c_versions";
+  static constexpr char kGrpcCMetadataHeapName[] = "grpc_c_metadata_buffer_heap";
+  static constexpr char kGrpcCHeaderEventDataHeapName[] = "grpc_c_header_event_buffer_heap";
+  static constexpr char kGrpcCEventDataHeapName[] = "grpc_c_event_buffer_heap";
+
+  inline static const std::map<std::string, enum grpc_c_version_t> kGrpcCMD5HashToVersion = {
```

Comment:
Done. I also added a todo (on your name because I didn't know whose name to put there) to add more hashes.
Another todo (not mentioned anywhere in the code) is obviously to support the versions in-between 1.19.0 and 1.41.1, and to support stripped binaries (as we discussed a while back).